keyboards/a_jazz/akc084/readme.md

```diff
@@ -0,0 +1,23 @@
+# A-JAZZ  AKC084
+
+A customizable 84keys keyboard
+
+![akc084](https://i.imgur.com/381vaD7.png)
+* Keyboard Maintainer: [Feng](https://github.com/fenggx-a-jazz)
+* Hardware Supported: [a-jazz](https://www.a-jazz.com)
+* Hardware Availability: [a-jazz](https://www.a-jazz.com)
```

Comment:
```suggestion
* Hardware Availability: [a-jazz]([ajazzstore.com/collections/all/products/ajazz-akp846](https://ajazzstore.com/collections/all/products/ajazz-akp846?variant=43770515587125))
```

Reply:
The linked hardware availability is a 6 row keyboard, but the profile updated shows for a 5 row. This doesn't seem to be the same keyboard.